openpype/tools/loader/widgets.py

```diff
@@ -584,14 +584,15 @@ def on_context_menu(self, point):
             for repre_doc in repre_docs:
                 repre_ids.append(repre_doc["_id"])
 
+                # keep only version ids without representation with that name
                 version_id = repre_doc["parent"]
-                if version_id not in version_ids:
-                    version_ids.remove(version_id)
+                version_ids.remove(version_id)
```

Comment:
I think it was meant as this because if the version id is not in the set it will crash.
```suggestion
                if version_id in version_ids:
                    version_ids.remove(version_id)
```

Reply:
Ah yes, you are right. Should we instead do [`set.discard()`](https://docs.python.org/3/library/stdtypes.html#frozenset.discard)? :) 

```python
a = set()
a.discard("b")
print(a)
# set()
a = set(["b"])
a.discard("b")
print(a)
# set()
```

Reply:
Much better.